### DISCUSSION TITLE
Encounter a segmentation fault when run gui.py

### DISCUSSION THREAD

**Discussion by ghost** [lukas-blecher/LaTeX-OCR#117] [Q&A]
When I run gui.py after I install all need in requirement.txt, I encountered a segmentation fault (core dumped). I use archlinux

**Comment by lukas-blecher** [comment]
might be related to #36 
I suggest you use the CLI and a third party snipping software. I don't know why these problems exist. Sorry

**Comment by ghost** [comment]
Thanks, I’ll try it later

发自我的iPad
